### ISSUE TITLE
Device specific support resource file not loaded

### ISSUE THREAD

**Issue by brlin-tw** [brlin-tw/iscan-snap#2]
```
[pid 34901] SYS_futex(0x7fa2ee417de8, 129, 0x7fffffff, 0) = 0
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd950951f0) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd950950e0) = -2
[pid 34901] SYS_stat("/etc/xml/catalog", 0x7ffd95094ef0) = 0
[pid 34901] SYS_openat(0xffffff9c, 0x55a708e4e0e7, 0, 0) = 11
[pid 34901] SYS_lseek(11, 0, 1)                  = 0
[pid 34901] SYS_read(11, "<?xml version="1.0"?>\n<!DOCTYPE catalog PUBLIC "-//OASIS//DTD XML Catalogs V1.0//EN"\n  "file:///usr/share/xml/schema/xml-core/catalog.dtd">\n<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">\n<delegatePublic publicIdStartString="+//IDN faq.org//D"..., 8192) = 3801
[pid 34901] SYS_read(11, "", 4391)               = 0
[pid 34901] SYS_close(11)                        = 0
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd950950e0) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095070) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095070) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095080) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095080) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095070) = -2
[pid 34901] SYS_stat("/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml", 0x7ffd95095070) = -2
[pid 34901] SYS_futex(0x7fa2ef33ef38, 129, 0x7fffffff, 0) = 0
```

**Comment by brlin-tw** [comment]
SANE_DEBUG_EPKOWA=CALL log:

```
cfg-obj.c:1563: [epkowa]{I} F/W name: 'Perfection V39'
model-info.c:248: [epkowa]{C} model_info_cache_get_model (Perfection V39)
model-info.c:345: [epkowa]{C} _model_info_cache_get_info (Perfection V39)
model-info.c:388: [epkowa]{C} _model_info_ctor (Perfection V39)
get-infofile.c:218: [epkowa][M] Unknown model name.
get-infofile.c:218: [epkowa][M] Unknown model name.
get-infofile.c:218: [epkowa][M] Unknown model name.
get-infofile.c:218: [epkowa][M] Unknown model name.
utils.c:90: [epkowa]{C} fw_name_to_hex 
model-info.c:571: [epkowa][m] failed to load external entity "/usr/local/share/iscan-data/device/50657266656374696f6e20563339.xml"
```

**Comment by brlin-tw** [comment]
```
brlin@brlin-mz-530:/home/brlin/文件/iscan-snap$ ltrace -f -e '-+xmlReadFile@libsane.so*' -s 256 iscan 2>&1 | tee ltrace.log
Gtk-Message: 08:35:36.021: Failed to load module "gail"
Gtk-Message: 08:35:36.031: Failed to load module "atk-bridge"
[pid 64730] libsane.so.1->xmlReadFile(0x556262fed300, 0, 96, 1) = 0
```

**Comment by brlin-tw** [comment]
Fixed by revision 86ec254.
